testsuite/layers/pom.xml

```diff
@@ -1319,6 +1371,58 @@
                             </configurations>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>web-clustering-provisioning</id>
+                        <goals>
+                            <goal>provision</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <install-dir>${layers.install.dir}/jpa</install-dir>
```

Comment:
should be web-clustering

Reply:
You should add the 2 new layers to all-layers-provisioning-full execution (end of this file).